keps/sig-network/2438-dual-stack-apiserver/README.md

```diff
@@ -0,0 +1,593 @@
+    - [Story 2 - APIServer Tooling](#story-2---apiserver-tooling)
+    - [Story 3 - Internal Clients](#story-3---internal-clients)
+  - [Notes/Constraints/Caveats (Optional)](#notesconstraintscaveats-optional)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [kube-apiserver Command-Line Arguments](#kube-apiserver-command-line-arguments)
+  - [Service and EndpointSlice Reconciling](#service-and-endpointslice-reconciling)
+  - [Test Plan](#test-plan)
+  - [Graduation Criteria](#graduation-criteria)
+  - [Upgrade / Downgrade Strategy](#upgrade--downgrade-strategy)
+    - [Handling of the &quot;<code>kubernetes</code>&quot; Service](#handling-of-the--service)
+    - [Handling of &quot;<code>kubernetes</code>&quot; Endpoints and EndpointSlices](#handling-of--endpoints-and-endpointslices)
+    - [Service vs EndpointSlice Sync Issues](#service-vs-endpointslice-sync-issues)
+  - [Version Skew Strategy](#version-skew-strategy)
+- [Production Readiness Review Questionnaire](#production-readiness-review-questionnaire)
+  - [Feature Enablement and Rollback](#feature-enablement-and-rollback)
+  - [Rollout, Upgrade and Rollback Planning](#rollout-upgrade-and-rollback-planning)
+  - [Monitoring Requirements](#monitoring-requirements)
+  - [Dependencies](#dependencies)
+  - [Scalability](#scalability)
+  - [Troubleshooting](#troubleshooting)
+- [Implementation History](#implementation-history)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [ ] (R) Graduation criteria is in place
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+The initial implementation of [dual-stack networking] in Kubernetes
+left the apiserver as a single-stack Service. This moves it to
+dual-stack (in dual-stack clusters).
+
+[dual-stack networking]: ../563-dual-stack/README.md
+
+## Motivation
+
+### Goals
+
+- Ensure kube-apiserver can listen dual-stack (subject to appropriate
```

Comment:
side comment. As we are moving `IPv6DualStack` feature to stable, you will probably need to define a new feature.